lib/SQLSyntax.lean

```diff
@@ -113,57 +115,59 @@ partial def mkEntry : Syntax → TermElabM Expr
   match stx with
   | `(propSymbol|=)  => pure $ if isEntry then `SQLProp.eqE else `SQLProp.eqC
   | `(propSymbol|<>) => pure $ if isEntry then `SQLProp.neE else `SQLProp.neC
+  | `(propSymbol|!=) => pure $ if isEntry then `SQLProp.neE else `SQLProp.neC
   | `(propSymbol|<)  => pure $ if isEntry then `SQLProp.ltE else `SQLProp.ltC
   | `(propSymbol|<=) => pure $ if isEntry then `SQLProp.leE else `SQLProp.leC
   | `(propSymbol|>)  => pure $ if isEntry then `SQLProp.gtE else `SQLProp.gtC
   | `(propSymbol|>=) => pure $ if isEntry then `SQLProp.geE else `SQLProp.geC
   | _                => throwUnsupportedSyntax
 
-partial def mkProp : Syntax → TermElabM Expr
-  | `(prop|TRUE)                              => mkConstM `SQLProp.tt
-  | `(prop|FALSE)                             => mkConstM `SQLProp.ff
+partial def elabProp : Syntax → TermElabM Expr
+  | `(prop|TRUE)                              => elabConst `SQLProp.tt
+  | `(prop|FALSE)                             => elabConst `SQLProp.ff
   | `(prop|$l:parsId $s:propSymbol $r:parsId) => do
-    mkAppM (← mkPropSymbol s false) #[← mkStrOfParsId l, ← mkStrOfParsId r]
+    mkAppM (← elabPropSymbol s false) #[← elabStrOfParsId l, ← elabStrOfParsId r]
   | `(prop|$c:parsId $s:propSymbol $e:entry)  => do
-    mkAppM (← mkPropSymbol s true) #[← mkStrOfParsId c, ← mkEntry e]
+    mkAppM (← elabPropSymbol s true) #[← elabStrOfParsId c, ← elabEntry e]
   | `(prop|$l:prop AND $r:prop)               => do
-    mkAppM `SQLProp.and #[← mkProp l, ← mkProp r]
+    mkAppM `SQLProp.and #[← elabProp l, ← elabProp r]
   | `(prop|$l:prop OR $r:prop)                => do
-    mkAppM `SQLProp.or #[← mkProp l, ← mkProp r]
+    mkAppM `SQLProp.or #[← elabProp l, ← elabProp r]
   | `(prop|NOT $p:prop)                       => do
-    mkAppM `SQLProp.not #[← mkProp p]
-  | `(prop|($p:prop))                         => mkProp p
+    mkAppM `SQLProp.not #[← elabProp p]
+  | `(prop|($p:prop))                         => elabProp p
   | _                                         => throwUnsupportedSyntax
 
-def mkJoin : Syntax → TermElabM Expr
-  | `(join|INNER) => mkConstM `SQLJoin.inner
-  | `(join|LEFT)  => mkConstM `SQLJoin.left
-  | `(join|RIGHT) => mkConstM `SQLJoin.right
-  | `(join|OUTER) => mkConstM `SQLJoin.outer
+def elabJoin : Syntax → TermElabM Expr
+  | `(join|INNER) => elabConst `SQLJoin.inner
+  | `(join|LEFT)  => elabConst `SQLJoin.left
+  | `(join|RIGHT) => elabConst `SQLJoin.right
+  | `(join|OUTER) => elabConst `SQLJoin.outer
   | _             => throwUnsupportedSyntax
 
-partial def mkFrom : Syntax → TermElabM Expr
+partial def elabFrom : Syntax → TermElabM Expr
   | `(sqlFrom|$t:ident)               => mkAppM `SQLFrom.table #[mkStrOfIdent t]
   | `(sqlFrom|$f:sqlFrom AS $t:ident) => do
-    mkAppM `SQLFrom.alias #[← mkFrom f, mkStrOfIdent t]
+    mkAppM `SQLFrom.alias #[← elabFrom f, mkStrOfIdent t]
+  | `(sqlFrom|$t₁:sqlFrom, $t₂:sqlFrom) => do mkAppM `SQLFrom.implicitJoin #[← elabFrom t₁, ← elabFrom t₂]
```

Comment:
This match needs to be adjusted accordingly with the new syntax to handle multiple tables

Reply:
Resolved by defining syntax differently: https://github.com/arthurpaulino/LeanMySQL/pull/3/commits/dd78f4d3e36b81f14b43f845e1c183cbf11e2aff